helpers/deproxy.py

```diff
@@ -448,6 +449,7 @@ def read_sized_body(self, stream):
         elif len(self.body) < size:
             tf_cfg.dbg(5, "Incomplete message received")
             raise IncompleteMessage()
+        self.parse_trailer(stream)
```

Comment:
I don't understand why it is needed? Please add a comment here. Only a chunked body has a trailer or not?

Reply:
tempesta will gather the chunked body from backend as fixed-length body, so we need to parse trailers here.
FYI, Transfer-Encoding is meaningless for HTTP/2, i.e. neither chunked or fixed-length, instead, it's chunked based on the fact of framing.